static/app/views/settings/project/server-side-sampling/serverSideSampling.tsx

```diff
@@ -72,18 +84,21 @@ export function ServerSideSampling() {
             'Server-side sampling provides an additional dial for dropping transactions. This comes in handy when your server-side sampling rules target the transactions you want to keep, but you need more of those transactions being sent by the SDK.'
           )}
         </TextBlock>
-        <PermissionAlert />
-        <Access organization={organization} access={['project:write']}>
-          {error && <LoadingError message={error} />}
-          {!error && loading && <LoadingIndicator />}
-          {!error && !loading && (
-            <RulesPanel
-              headers={['', t('Operator'), t('Condition'), t('Rate'), t('Active'), '']}
-            >
-              <Promo />
-            </RulesPanel>
+        <PermissionAlert
+          access={['project:write']}
```

Comment:
noticed the same thing https://github.com/getsentry/sentry/blob/b78a5c7789210f157062791859b2df80f97b202f/static/app/views/settings/project/server-side-sampling/serverSideSampling.tsx#L96 :)